modules/xray/shared/src/main/scala/natchez/xray/XRaySpan.scala

```diff
@@ -108,6 +119,8 @@ private[xray] final case class XRaySpan[F[_]: Sync](
 
 private[xray] object XRaySpan {
 
+  private[XRaySpan] val keyRegex: Regex = """[^A-Za-z0-9_]""".r
```

Comment:
Regex is also fiddly on JS but this seems innocent enough :)

Reply:
I'm definitely open to suggestions. (Basically it needs to find characters that aren't alphanumeric or the underscore so they can be replaced when used as annotation keys for XRay.)

Reply:
Eh, probably fine as is :) I've seen some weird stuff before where you need to enable ES2018 otherwise regexes just fail silently, but Node.js lambdas probably support ES2018 anyway if it turns out to be necessary.